CONTRIBUTING.md

```diff
@@ -216,6 +216,7 @@ For all resources, if the environmental variable is not provided, a new instance
 | `azext_dt_ep_eventhub_topic`	| Digital Twin Data History and Endpoint Tests	| The Event Hub instance in the Event Hub Namespace to use.	|
 | `azext_dt_ep_eventhub_topic_consumer_group`	| Digital Twin Data History Tests	| The Event Hub consumer group name to use. Defaults to "$Default".	|
 | `azext_dt_ep_rg`	| Digital Twin Data History and Endpoint Tests	| The resource group that contains the endpoint (Event Hub, Event Grid, Service Bus) variables. If not provided, `az_iot_testrg` will be used as the resource group.	|
+| `azext_dt_max_model_per_batch`	| Digital Twin Model Tests	| The maximum value of process models in batches if models to process exceed the API limit. Defaults to 30.	|
```

Comment:
Description suggestion: "The maximum number of models per batch to submit to the DT Instance if the total set of models exceed the single page API limit."